src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/RecallAuditComponentTest.kt

```diff
@@ -2,65 +2,87 @@ package uk.gov.justice.digital.hmpps.managerecallsapi.component
 
 import com.natpryce.hamkrest.assertion.assertThat
 import com.natpryce.hamkrest.equalTo
+import org.assertj.core.api.AbstractOffsetDateTimeAssert
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.within
 import org.junit.jupiter.api.Test
-import org.springframework.beans.factory.annotation.Autowired
-import uk.gov.justice.digital.hmpps.managerecallsapi.db.Recall
+import uk.gov.justice.digital.hmpps.managerecallsapi.controller.BookRecallRequest
+import uk.gov.justice.digital.hmpps.managerecallsapi.controller.UpdateRecallRequest
+import uk.gov.justice.digital.hmpps.managerecallsapi.domain.FieldName
 import uk.gov.justice.digital.hmpps.managerecallsapi.domain.FirstName
 import uk.gov.justice.digital.hmpps.managerecallsapi.domain.FullName
 import uk.gov.justice.digital.hmpps.managerecallsapi.domain.LastName
 import uk.gov.justice.digital.hmpps.managerecallsapi.domain.NomsNumber
 import uk.gov.justice.digital.hmpps.managerecallsapi.domain.PrisonId
-import uk.gov.justice.digital.hmpps.managerecallsapi.domain.RecallId
-import uk.gov.justice.digital.hmpps.managerecallsapi.domain.random
-import uk.gov.justice.digital.hmpps.managerecallsapi.service.RecallAuditService
-import java.time.Instant
 import java.time.OffsetDateTime
-import java.time.ZoneId
+import java.time.temporal.ChronoUnit
 
 class RecallAuditComponentTest : ComponentTestBase() {
 
   private val nomsNumber = NomsNumber("123456")
 
-  @Autowired private lateinit var recallAuditService: RecallAuditService
-
   @Test
   fun `get currentPrison audit for a recall that has been booked`() {
-    val recallId = ::RecallId.random()
-    val createdByUserId = authenticatedClient.userId
-
-    val now = OffsetDateTime.ofInstant(Instant.parse("2021-10-04T14:15:43.682078Z"), ZoneId.of("UTC"))
+    val savedRecall = authenticatedClient.bookRecall(BookRecallRequest(nomsNumber, FirstName("Brian"), null, LastName("Badgering")))
```

Comment:
Might be clearer if this person name is e.g. "The Offender".  Or if we switch from "Bertie Badger" to "Current User"?

Reply:
Yep, probably a good change at some point. Cassie Caseworker and Ollie Offender?